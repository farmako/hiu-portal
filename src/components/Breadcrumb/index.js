import Breadcrumb from './Breadcrumb.view';

export default Breadcrumb;
