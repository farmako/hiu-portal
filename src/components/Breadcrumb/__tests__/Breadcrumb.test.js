import React from 'react';
import { shallow } from 'enzyme';
import Breadcrumb from '../Breadcrumb.view';

describe('Breadcrumb', () => {
  it('should render properly', () => {
    global.BASE_NAME = '/';
    const wrapper = shallow(<Breadcrumb />);
    expect(wrapper).toMatchSnapshot();
  });
});
