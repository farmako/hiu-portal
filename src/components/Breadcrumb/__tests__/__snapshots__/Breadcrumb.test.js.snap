// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb should render properly 1`] = `
<WithStyles(ForwardRef(Breadcrumbs))
  aria-label="breadcrumb"
  className="makeStyles-container-1"
>
  <WithStyles(ForwardRef(Link))
    color="inherit"
    href="/"
  >
    Consent List
  </WithStyles(ForwardRef(Link))>
  <WithStyles(ForwardRef(Typography))
    color="textPrimary"
  >
    Health Data
  </WithStyles(ForwardRef(Typography))>
</WithStyles(ForwardRef(Breadcrumbs))>
`;
